vector/src/main/java/fr/gouv/tchap/adapters/TchapContactAdapter.java

```diff
@@ -395,10 +395,26 @@ private void populateViews(final ParticipantAdapterItem participant, final int p
                 vContactName.setTypeface(null, Typeface.BOLD);
             }
 
+            // display the participant's avatar
             participant.displayAvatar(mSession, vContactAvatar);
 
+            // display the participant's name
             vContactName.setText(DinsicUtils.getNameFromDisplayName(participant.mDisplayName));
-            vContactDomain.setText(DinsicUtils.getDomainFromDisplayName(participant.mDisplayName));
+
+            // display the participant's domain
+            String domainName = DinsicUtils.getDomainFromDisplayName(participant.mDisplayName);
+
+            if (null == domainName || domainName.isEmpty()) {
+                String emailAddress = participant.mContact.getEmails().get(0);
+                String[] components2 = emailAddress.split("@");
+
+                if (components2.length>1) {
+                    String domain = components2[1].substring(0,components2[1].indexOf("."));
+                    String formattedDomain = domain.substring(0, 1).toUpperCase() + domain.substring(1);
```

Comment:
This is a very good point to uppercase the first letter. 
You have to check that the domain string length is higher than 2 ;)